end_to_end/tpu/mistral/7b/test_mistral-7b.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# This file runs on daily basis and demonstrates:
```

Comment:
nit: mention where we run this test. v4-8 in this case.

Reply:
Done